mmdet/utils/deployment/symbolic.py

```diff
@@ -20,6 +20,49 @@
 from torch.onnx.symbolic_registry import register_op, get_registered_op, is_registered_op
 
 
+def convert_args_and_output(op_name=None):
+    def flatten(args):
+        flat_args = []
+        for a in args:
+            if isinstance(a, (list, tuple)):
+                flat_args.extend(flatten(a))
+            else:
+                flat_args.append(a)
+        return flat_args
+
+    def convert_to_cpu(args):
+        cpu_args = []
+        for a in args:
+            if isinstance(a, (list, tuple)):
+                cpu_args.extend(convert_to_cpu(a))
+            elif isinstance(a, torch.Tensor):
+                cpu_args.append(a.cpu())
+            else:
+                cpu_args.append(a)
+        return cpu_args
+
+    def as_list(x):
+        if not isinstance(x, (tuple, list)):
+            return [x, ]
+        return x
+    
+    def decorator(func):
+        @wraps(func)
+        def wrapped_function(*args):
+            if op_name == 'roi_feature_extractor':
```

Comment:
Let's avoid using any custom logic here. Whatever is needed for some particular ad hoc use case, should be moved to a stand alone wrapper/decorator/function.

Reply:
I have moved the logic for handling this case into a function.